cns/restserver/util.go

```diff
@@ -413,7 +413,7 @@ func (service *HTTPRestService) getAllNetworkContainerResponses(
 		}
 		nmaNCs := map[string]string{}
 		for _, nc := range ncVersionListResp.Containers {
-			nmaNCs[cns.SwiftPrefix+nc.NetworkContainerID] = nc.Version
+			nmaNCs[cns.SwiftPrefix+strings.ToLower(nc.NetworkContainerID)] = nc.Version
```

Comment:
let's lowercase the _whole_ string

Reply:
Hi Evan, the ncid is formed by Swift_NCID; so we just need to lowercase NCID part; cns.SwiftPrefix is a fixed string

Reply:
I understand that, but we should treat the whole map key as case insensitive instead of having special handling of the partial string.